cg/store/api/find_business_data.py

```diff
@@ -185,6 +185,22 @@ def get_cases_from_sample(self, sample_entry_id: str) -> Query:
             case_samples=self._get_join_case_sample_query(),
         )
 
+    def get_samples_by_case_and_pipeline(
```

Comment:
Is it really necessary to filter by pipeline? Why is it not enough to just filter by case_id? Can the case have samples being run in other pipelines?